src/palakSJun20/ArrayIntersectionMetho2.java

```diff
@@ -0,0 +1,78 @@
+/*Program 2: Find intersection(common elements) of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+
+Output: 10.45, 18.35
+Method Signature : double[] getIntersectionOfArray(double[] arr1, double[] arr2)
+
+Approach 2 : Finding common between 2 arrays and insert unique element in another array 
+ */
+package palakSJun20;
+
+import java.util.Arrays;
+
+public class ArrayIntersectionMetho2 {
+	int count = 0;
+	int resultArrayCount = 0;
+
+	void findIntersectionOfArray(double[] array1, double[] array2) {
+		int size = 0;
+		if (array1.length > array2.length)
+			size = array1.length;
```

Comment:
findIntersectionOfArray in this method here you should create min length array. You are creating max length array.